### PR TITLE
add ky and ru languages story #3875

### PR DESCRIPTION
**Pull request by phillimorland**
…languages, add nl, ky, ru as cms simple page languages, add new translation class for legal pages story #3875

### CODE REVIEW

File: adhocracy-plus/config/settings/base.py [hunk 1]
```diff
@@ -200,12 +200,14 @@
...
-    ('nl', _('Dutch'))
+    ('nl', _('Dutch')),
+    ('ky', _('Kyrgyz')),
+    ('ru', _('Russian'))
 ]
 
 PARLER_LANGUAGES = {
     1: [{'code': language_code} for language_code, language in LANGUAGES],
-    'default': {'fallbacks': ['en', 'de'],
+    'default': {'fallbacks': ['en', 'de', 'nl,' 'ky', 'ru'],
```
Comment (phillimorland) [inline adhocracy-plus/config/settings/base.py:210]: they are not needed here right?

**Comment by phillimorland** [comment]
I'm not sure if theres more to it then this until we have the transifix? if so just let me know, oh and I wasn't sure on the paler languages so left a question.

File: adhocracy-plus/config/settings/base.py (hunk 1 above)
Comment (fuzzylogic2000) [reply adhocracy-plus/config/settings/base.py:210]: Yes, I think so, for the organsiation languages.

**Review by fuzzylogic2000 — APPROVED** [review]
Yeah!

**Pull request merged**